TwilioVerifySDK/TwilioSecurity/Sources/Keychain/KeychainQuery.swift

```diff
@@ -111,10 +111,9 @@ struct KeychainQuery: KeychainQueryProtocol {
   
   func delete(withKey key: String) -> Query {
     [kSecClass: kSecClassGenericPassword,
-     kSecAttrAccount: key,
-     kSecAttrAccessible: kSecAttrAccessibleAfterFirstUnlockThisDeviceOnly] as Query
```

Comment:
Why are we removing `kSecAttrAccessible`?

Reply:
It wasn't able to delete the item from they keychain when using the `kSecAttrAccessible`, always returned notFound instead of success. And when using the get I was able to found it. With this fix it is capable of found the key and delete the item.

Reply:
Does this apply for biometrics protected entries or all? I don't want to impact another storage like factors storage in the verify SDK

Reply:
This should work for any keychain item, or any item using a `kSecClass` of `kSecClassGenericPassword` & a `key`. 

Apple documentation about those queries are very vague, but according to apple when using more attributes in the query, is possible to encounter weird behaviors.
```
If more than one of these return keys is specified, the behavior is undefined.
```
https://developer.apple.com/documentation/security/1395547-secitemdelete?language=objc

The query itself should be able to find protected and not protected items in the keychain with the specified parameters, and it can be used to delete a keychain item, the system itself can request authorization if needed in any of the keychain functions.

Reply:
@AlejandroOrozco Just to be sure, could you create a factor using the sample app version from main branch and try to delete that factor using this version from biometrics branch? I would like to validate that deleting a factor keeps working

Reply:
@yafuquen  Yep, it is working flawlessly.
![Screen Shot 2021-10-05 at 6 02 37 PM](https://user-images.githubusercontent.com/4898461/136113720-9044c5f3-ac88-4524-a557-3da66d14fbfa.png)

I also found that we have a `delete(withAlias)`, so I may update both to keep parity, what do you think?